entries/git-workflow-guidelines/commits.md

```diff
@@ -0,0 +1,213 @@
+
+![How Git tracks file changes](../images/file-tracking-in-git.png)
+
+*Unmodified* files includes all tracked files that haven't been created or modified since the last commit. As soon as you save a new file or save an edited file, Git tracks that as being *modified*. When you are happy with your changes, you can *stage* the modifications by adding them to the *staging area*, which includes all changes that will be a part of the next commit. Adding changes to the staging area can, therefore, be said to be the first phase of a commit. In the second phase, you commit the staged changes, and Git saves a snapshot of your repository with these changes. This snapshot includes only the staged changes and the unmodified changes. Changes that haven't been staged are kept but not included in the snapshot.
+
+Note that you can add as many files and changes to the staging area as you want before you commit them. All changes in the staging area at the time of commit will be a part of the same commit. In this way, the staging area allows you to control which changes you want to include in a specific commit, so you can collect the changes connected to the same unit of work in one commit (we will return to what that means in the section [Atomic Commits](#atomic-commits) below).
+
+Commits are always created locally in the specific branch you are currently on. Therefore, it is important to check that you are on the right branch before you commit (check the [Branching](branching.md#branching-in-practice) post for how you check which branch you are currently on and how to change branch). After you have created your commits, they needs to be *pushed* to the remote repository (on GitHub or or any other version control service that uses Git) for anyone else to be able to see it.
+
+After you have pushed your commits, your work will be backed up on the remote storage. As your work progresses, you continuously commit and push changes to the branch you are working on until you are ready to ask for feedback through a pull request (more on pull requests and the review process in the [Pull Request](prs.md) post).
+
+## Atomic Commits
+
+Now you should have an understanding of that commits are, but how often should you commit, and how do you choose what to include in a commit?
+
+Ideally, you should commit often, based around isolated, complete changes, a practice called **atomic** commits. This means that each commit should document a *single, complete unit of work* that represents a specific idea (as described in the post [Make Atomic Git Commits](https://www.aleksandrhovhannisyan.com/blog/atomic-git-commits/#atomic-commits-and-the-single-responsibility-principle) and the [Git guides](https://github.com/git-guides/git-commit)). As a result, that one commit should do one - and only one - thing, which can be summed up in a short, simple sentence (i.e., in the commit message).
```

Comment:
move to a reference list in the bottom of the post?

Reply:
I personally prefer them in the text itself.

Reply:
Also written out like this? I could refer to them with numbers in the text, like I have done in the github flow decision post (see screenshot below)? 
![Screenshot 2023-11-21 at 10 16 50](https://github.com/seedcase-project/team/assets/40836345/49da5435-345d-448b-8f7b-08e78ce57408)

And then it looks like this in the bottom of the page:
![Screenshot 2023-11-21 at 10 17 49](https://github.com/seedcase-project/team/assets/40836345/1c887466-7a97-4a50-9848-e56a9943e87c)